app/controllers/application_controller.rb

```diff
@@ -1,3 +1,13 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
+
+  before_filter :configure_permitted_parameters, if: :devise_controller?
+
+  protected
+
+    def configure_permitted_parameters
+      devise_parameter_sanitizer.permit(:sign_up) { |u| u.permit(:name, :email, :password) }
```

Comment:
If you foresee that in the future you might want to have `:first_name` and `:last_name` attributes on `User`, instead of `:name`, it will be much easier to make that switch now.

Not a recommendation, per se, just something to consider.